logging/logger.go

```diff
@@ -96,6 +99,7 @@ func NewLogger(ctx context.Context, cfg LoggerConfig) zerolog.Logger {
 	multiWriter := zerolog.MultiLevelWriter(outputs...)
 	logger := zerolog.New(multiWriter)
 	logger = logger.With().Timestamp().Logger()
+	logger = logger.With().Any("group", cfg.Name).Logger()
```

Comment:
I suppose we can use `Str` here to avoid JSON marshaling of `Any`, since we already know that `cfg.Name` is a string. WDYT?
```suggestion
	logger = logger.With().Str("group", cfg.Name).Logger()
```